server/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -55,16 +151,28 @@ public ClusterHealthResponse(String clusterName, String[] concreteIndices, Clust
     }
 
     public ClusterHealthResponse(String clusterName, String[] concreteIndices, ClusterState clusterState, int numberOfPendingTasks,
-                                 int numberOfInFlightFetch, int delayedUnassignedShards, TimeValue taskMaxWaitingTime) {
+        int numberOfInFlightFetch, int delayedUnassignedShards, TimeValue taskMaxWaitingTime) {
```

Comment:
Can you add the indent back? It is much easier to read if the parameters don't line up with the code.